### ISSUE TITLE
[a11y]: Popover [tab tip] - expand/collapsed status not announced by SR

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon#14824]
### Package

@carbon/react

### Browser

FF, Safari & iPhone

### Package version

@carbon/react@1.38.0

### Automated testing tool and ruleset

Manual QA

### Assistive technology

VO & JAWS

### Description
Screen reader does not announce the expanded/collapsed status. Hence the user would not know the current status

### Recommended Fix
Use aria-expanded attribute and update the value according to the toggle status

When the state of an element i.e. disabled, selected, expanded, currently active etc. is not conveyed for the screen reader user, it can be misleading for the user. . this impacts the user interaction.

### WCAG 2.1 Violation

4.1.2 Name, Role, Value
